test/parallel/test-repl-history-navigation.js

```diff
@@ -554,6 +556,27 @@ const tests = [
     expected: [],
     clean: false
   },
+  {
+    env: { NODE_REPL_HISTORY: defaultHistoryPath },
+    test: [
+      'const util = {}',
+      ENTER,
+      'proc',
+      RIGHT,
+      SIGINT
+    ],
+    expected: [
+      prompt,
+      ...'const util = {}'.split(''),
```

Comment:
It would be more readable if you break lines only when the REPL does. Also the `split` is not useful here.
```suggestion
      prompt, ...'const util = {}',
```